pkg/allow_list/github_test.go

```diff
@@ -0,0 +1,116 @@
+package allow_list
+
+import (
+	"testing"
+)
+
+func TestIsGithubRepoAllowed(t *testing.T) {
+	type args struct {
+		fullName string
+		allowEnv string
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "org/repo_allowed",
+			args: args{
+				fullName: "org/repo",
+				allowEnv: "org,other_org",
+			},
+			want: true,
+		},
```

Comment:
If you wanted, you could make both structs anonymous:

```suggestion
	tests := []struct {
		name string
		args struct {
			fullName string
			allowEnv string
		}
		want bool
	}{
		{
			name: "org/repo_allowed",
			args: {
				fullName: "org/repo",
				allowEnv: "org,other_org",
			},
			want: true,
		},
```

Reply:
I'm going to revert to named struct so I don't have to define it for each case eg. - unless there's something I'm missing?

```
		{
			name: "different_org/repo_denied",
			args: struct {
				fullName string
				allowEnv string
			}{
				fullName: "different_org/repo",
				allowEnv: "org,other_org",
			},
			want: false,
		},
```